server/app/routes/cart/index.js

```diff
@@ -4,18 +4,25 @@ var router = require('express').Router();
 module.exports = router;
 
 var cartModel = mongoose.model('Cart');
+var lineItemModel = mongoose.model('LineItem')
 
 //GET CART
 
 router.get('/', function (req, res) {
 	var sessionId = req.sessionID;
 
-	cartModel.findOne({sessionId: sessionId}).populate('films').exec(function (err, cart) {
+	var opts = {
+	            path: 'film',
+	        };
+
+	cartModel.findOne({sessionId: sessionId}).exec(function (err, cart) {
 		if(err) throw err;
 
-		console.log(cart);
+		lineItemModel.populate(cart.lineItems, opts, function(err, populatedLineItem) {
```

Comment:
This is cool - subpopulating embedded Mongo documents took me like 3 hours to figure out, but the great old Internet finally came to the rescue.